spring-cloud-task-docs/src/main/asciidoc/batch-starter.adoc

```diff
@@ -0,0 +1,532 @@
+
+[[batch-job-starter]]
+= Single Step Batch Job Starter
+
+[[partintro]]
+--
+This section goes into how to develop a Spring Batch `Job` with a single `Step` using the
+starter included in Spring Cloud Task. This starter allows users to use configuration
+to define an `ItemReader`, an `ItemWriter`, or a full single step Spring Batch `Job`.
+To read more about Spring Batch and its capabilities, read its documentation
+https://spring.io/projects/spring-batch[here].
+--
+
+To obtain the starter, add the following to your build:
+
+Maven:
+[source,xml]
+----
+<dependency>
+    <groupId>org.springframework.cloud</groupId>
+    <artifactId>spring-cloud-starter-single-step-batch-job</artifactId>
+    <version>2.3.0-SNAPSHOT</version>
+</dependency>
+----
+
+Gradle:
+[source,groovy]
+----
+compile "org.springframework.cloud:spring-cloud-starter-single-step-batch-job:2.3.0-SNAPSHOT"
+----
+
+[[job-definition]]
+== Defining a Job
+
+You can use the starter to define as little as an `ItemReader` or as much as a full `Job`.
```

Comment:
`ItemReader`, `ItemWriter` or as much as a full `Job`.

Reply:
There should be a comma after `ItemWriter` if it gets added to the list.